### PR TITLE
add a configuration parameter to allow skipping the sync of VM notes

### PR DESCRIPTION
**Pull request by gstorme**
When using Veeam backup, the VM notes are updated with the backup job details.
These contain a date field, so it's updated frequently.

I've added a config parameter that allow skipping the sync of these notes into the comments field in netbox.

### CODE REVIEW

File: module/sources/vmware/connection.py [hunk 1]
```diff
@@ -1940,7 +1942,7 @@ def add_virtual_machine(self, obj):
 
         if platform is not None:
             vm_data["platform"] = {"name": platform}
-        if annotation is not None:
+        if bool(self.skip_vm_comments) is False and annotation is not None:
```
Comment (bb-Ricardo) [inline module/sources/vmware/connection.py:1945]: This should stay unchanged. Just add an `annotation = None` above line 1922

**Review by bb-Ricardo — CHANGES_REQUESTED** [review]
The idea is great. Can you modify the change as suggested?

thank you.

File: module/sources/vmware/connection.py (hunk 1 above)
Comment (gstorme) [reply module/sources/vmware/connection.py]: Like this? 8c2bdf65ad7a82ae797b1e024e4c66cd259eab82

**Review by bb-Ricardo — APPROVED** [review]
Excellent, Thank you.

**Pull request merged**